scripts/create-dmg.sh

```diff
@@ -0,0 +1,636 @@
+#!/usr/bin/env bash
+
+# copied over from https://raw.githubusercontent.com/create-dmg/create-dmg/refs/heads/master/create-dmg
```

Comment:
You're violating the license of the repo:
```
The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.
```
https://github.com/create-dmg/create-dmg/blob/master/LICENSE

Reply:
sorry for that, fixed.